_transformations/scale/readme.md

```diff
@@ -14,17 +14,43 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges in a scale file.
+The input string must be in numerical format.
 
 The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
 
 ## Example
 
-transform/temps.scale:
+The following example shows how to break down numeric UV values into fixed UV index categories.
 
-```properties
-[12..23]=Temp between 12 or higher and 23 or lower
-]12..23[=Temp over 12 and below 23
-..23]=Any temp up to and including 23
-]12..=Any temp above 12
+example item:
+
+```java
+Number Uv_Sensor_Level "UV Level [SCALE(uvindex.scale):%s]"
+```
+
+uvindex.scale in transform folder:
+
+```
+[0,3]=1
+[3,6]=2
+[6,8]=3
+[8,10]=4
+[10,100]=5
 ```
```

Comment:
Which is it, `[10,100]` or `[10..100]`

Also how about showing the "smaller than 3" and "bigger than 10" logic from below?

Reply:
I will check that in my installation and rewrite it then.

For the other logic, i will comment below.